s/sentry-sdk/sentry-sdk_ubi_8.4.sh

```diff
@@ -0,0 +1,65 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : sentry-sdk
+# Version       : 0.14.3,0.14.4,0.18.0,0.19.5,1.0.0
+# Source repo   : https://github.com/getsentry/sentry-python
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Kishor Kunal Raj <kishor.kunal.mr@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "sentry-sdk" ] ; then
+  rm -rf sentry-sdk
+fi
+
+# Dependency installation
+dnf update
+dnf install -y git python36 make python3-devel gcc gcc-c++
+ln -s /usr/bin/python3 /usr/bin/python
+
+# Download the repos
+git clone https://github.com/kishorkunal-raj/sentry-python.git
+
+
+# Build and Test rfc3986-validator
```

Comment:
what is this to do with rfc3986-validator

Reply:
@gururajrkatti, It was a wrong comment added, corrected it.